introduction.md

```diff
@@ -1,2 +1,7 @@
 #hello I am part of this dojo! My favorite course path is the database path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?